src/hammer-tech/filters.py

```diff
@@ -167,6 +167,21 @@ def sort_func(lib: "Library"):
         return LibraryFilter.new("lef", "LEF physical design layout library", is_file=True, filter_func=filter_func,
                                  paths_func=paths_func, sort_func=sort_func)
 
+    @property
+    def verilog_sim_filter(self) -> LibraryFilter:
+        """
+        Select verilog sim files for gate level simulation
+        """
+
+        def filter_func(lib: "Library") -> bool:
+            return lib.verilog_sim is not None
+
+        def paths_func(lib: "Library") -> List[str]:
+            assert lib.verilog_sim is not None
+            return [lib.verilog_sim]
+
+        return LibraryFilter.new("verilog_sim", "VSIM gate level sim verilog sources", is_file=True, filter_func=filter_func, paths_func=paths_func)
```

Comment:
I'm not sure I understand why this is a library filter- I was thinking this would be implemented using the inputs/output interface like how syn passes the mapped netlist to PAR

Reply:
Talked about this offline, the library filter is fine